index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/styles.css">
+
+</head>
+<body>
+    
+
+    <header class="header">
+        <div class="header-links">
+            <ul>
+                <li>
+                    <i id="facebook-icon" class="fa fa-facebook"></i>
+                </li>
+                <li>
+                    <i id="twitter-icon" class="fa fa-twitter"></i>
+                </li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <div class="headline-section-nav">   
+            <a href="index.html" class="link logo"><img src="images/cc-logo-dark.png" class="logo-img" alt="logo"></a>
+            <button class="hamburger"><i class="fa fa-bars"></i></button>
+            <nav class="">
+                <ul class="mainMenu d-none">
+                    <li class="closeMenu">&times;</li>
+                    <li><a href="#" class="nav-links">Home</a></li>
+                    <li><a href="about.html" class="nav-links">About</a></li>
+                    <li id="nav-link-border">CC Campaign</li>
+                
+                </ul>
+            </nav>
```

Comment:
- [ ] Please make sure that the `Navbar` in mobile version has the same color as the headline section, I encourage you to do that 
<img width="446" alt="Screen Shot 2022-07-28 at 13 26 42" src="https://user-images.githubusercontent.com/61790587/181504849-5a4f70fd-4196-4e77-a052-3e23a1d68458.png">